scripts/dockerfiles/run.sh

```diff
@@ -85,6 +90,65 @@ done
         /var/vcap/jobs/{{ $job.Name }}/bin/run
     {{ end }}
 {{ else }}
+    # Run all the scripts called post-start, if any.
+    function post-start-paths()
+    {
+	local fnames=""
+	for fname in "$(find /var/vcap/jobs/*/bin -name post-start)" ; do
+	    fnames="${fnames} ${fname}"
+	done
+	echo "${fnames}"
```

Comment:
Looking at it again, why do we even wrap it in a variable? Isn't this method equivalent to running `find` directly?